client/src/pages/Prospects.js

```diff
@@ -60,20 +62,75 @@ const useStyles = makeStyles((theme) => ({
 
 const Prospects = () => {
   const classes = useStyles();
+  const [prospects, handleProspects] = useState([]);
+  const [dialog, handleDialog] = useState(null);
+  const [campaigns, handleCampaigns] = useState(null);
+  const [campaignId, setCampaignId] = useState(null);
+
+  useEffect(() => {
+    getAllCampaigns();
+  }, [])
+
+  // Need to replace hard code user with JWT token 
+  const user = 1
+
+  const getAllCampaigns = () => {
+    fetch(`http://localhost:5000/campaigns/${user}`)
+    .then(res => res.json())
+      .then(data => {
+        handleCampaigns(data.Campaigns)
+    })
+    .catch(err => {
+      console.log(err.message);
+    });
+  }
+
+  const callSave = () => {
+    handleDialog(false)
+    saveProspectsToCampaign()
+  } 
+
+  const saveProspectsToCampaign = () => {
+    const data = {
+      "prospect_ids": prospects,
+    };
 
-  const prepareData = () => {
+    fetch(`http://localhost:5000/campaign/${campaignId}/prospects`, {
```

Comment:
we don't need to have localhost:5000 because of the proxy - can use a relative route - and also reach out to Ben for the utils for sending it with Auth